website/docs/developer/research/research-deep-glacier-migration.mdx

```diff
@@ -0,0 +1,242 @@
+description: Research notes on deep archive migration.
+---
+import MyImage from '@site/docs/templates/pan-zoom-image.mdx';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## Overview
+
+The goal of this research is to provide an initial design and recommendation for migrating a customer's ORCA Flexible Retrieval (formerly Glacier) archive to Deep Archive storage class. Deep archive S3 storage type is the cheapest glacier storage type currently present in AWS S3. For archive data that does not require immediate access such as backup or disaster recovery use cases, choosing `Deep Archive Access` tier could be a wise choice.
+- Storage cost is ~$1/TB data per month. 
+- Objects in the Deep Archive Access tier can be restored to the Frequent Access tier within 12-48 hours depending on the recovery type. Once the object is in the Frequent Access tier, users can send a GET request to retrieve the object. Like Flexible retrieval, the restored file is only available in frequent access for a configured number of days. For more information on retrieving objects from Flexible and Deep Archive storage, see the [documentation](See https://docs.amazonaws.cn/en_us/AmazonS3/latest/userguide/restoring-objects-retrieval-options.html
+).
+
+### Assumptions
+
+- List of objects already exists in orca catalog and S3 bucket.
+- Users should be able to provide some information(possibly a collection or a combination of collection and granuleId) to migrate objects.
+- A serverless approach will be used utilizing several Python lambda functions, SNS/SQS and S3 bucket event notifications.
+- The glacier bucket has versioning enabled.
+
+### Use Cases
+
+Based on the discussion with ORCA team, we came up with two use cases that the end user might be interested in:
+- Convert all the holdings or holdings with a known key prefix from Flexible Retrieval(formerly Glacier) to Deep Archive.
+- Convert specific collection or granules from Flexible Retrieval(formerly Glacier) to Deep Archive.
+
+The details of each use cases and relevant architecture drawings can be found in the next sections.
+
+### Migrating all ORCA holdings to deep archive.
+
+The migrate all holdings use case assumes that the end user wants to migrate all ORCA holdings to the DEEP ARCHIVE storage type or specific collections of data that can be grouped together via a key path like `/ShortName/Version` within the ORCA bucket. The most efficient and cost effective approach to this use case would to have the operator perform the migration steps manually as outlined below.
+
+The steps for performing the migration are as follows:
+1. Configure ORCA default storage type or the collection level storage type overide to DEEP ARCHIVE and re-deploy ORCA if needed. See this [documentation](https://github.com/nasa/cumulus-orca/blob/develop/CHANGELOG.md#migration-notes) for additional details.
+2. Run the lifecycle configuration with transition_days = 0 as shown in the terraform code example below.
```

Comment:
This is not a viable instruction, as it does not detail where the code block is placed.

Reply:
If this is meant to be just an example, and not a recommendation on using TF vs. AWS GUI, then I'd recommend calling that out.

Reply:
based on the architecture drawing, it will be performed from aws console.

Updated the sentence to 
`2. From the AWS console, set the lifecycle configuration with transition_days = 0 as shown in the terraform code example below.`